Source/OxyPlot.Pdf.Tests/OxyPlot.Pdf.Tests.csproj

```diff
@@ -1,19 +1,18 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
-        <TargetFramework>net462</TargetFramework>
+        <TargetFrameworks>net462</TargetFrameworks>
         <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
         <Description>OxyPlot Pdf unit tests</Description>
-        <PackageLicenseExpression>MIT</PackageLicenseExpression>
-        <Copyright>OxyPlot contributors</Copyright>
         <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-        <NoWarn>CS0618</NoWarn>
+        <NoWarn>CS0618,NU1701</NoWarn>
```

Comment:
OK for now I think; we should investigate PDF Sharp again: looks like a net6.0 package was released 3 month ago.